### PR TITLE
create vente-privee invoices fetch and fix some translations

### PR DESCRIPTION
**Pull request by SomeAverageDev**
This PR adds the new connector "vente-privee.com", which fetchs invoices from the website shop.
Moreover, I modified some translations : I changed the word "bill" to "invoice" as it seems to be more accurate.
I also updated the sfr connectors to fix a small translation ID error
But If I am wrong, I can change it back...

### CODE REVIEW

**Comment by frankrousseau** [comment]
This PR contains many changes. Could you explain them and tell what problems do they solve?

It's something mentioned in the contributing rules: https://github.com/cozy-labs/konnectors#contributing-rules

**Comment by frankrousseau** [comment]
By the way, thank you for this new konnector!

**Comment by SomeAverageDev** [comment]
hello @frankrousseau,
You are right, sorry. I updated the description.
I change the translation files more that I needed to. If it's not appropriate, don't hestitate to tell me and I will change it back.

**Comment by frankrousseau** [comment]
Most websites uses the word bill. Can you fix this in another PR?

File: server/konnectors/venteprivee.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+const saveDataAndFile = require('../lib/save_data_and_file');
+const linkBankOperation = require('../lib/link_bank_operation');
+
+const log = require('printit')({
+  prefix: 'vente-privee',
+  date: true,
+});
+
+const fileOptions = {
+  vendor: 'Vente-Privee',
```
Comment (frankrousseau) [inline server/konnectors/venteprivee.js]: don't use Uppercases in file names.

File: server/konnectors/venteprivee.js [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+  dateFormat: 'YYYYMMDD',
+};
+
+const Bill = require('../models/bill');
+const baseUrl = 'https://secure.fr.vente-privee.com';
+
+// Konnector
+const connector = module.exports = baseKonnector.createNew({
+  name: 'Vente-Privee.com',
+  slug: 'vente-privee-com',
```
Comment (frankrousseau) [inline server/konnectors/venteprivee.js]: Don't set slug yourself.

File: server/konnectors/venteprivee.js [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+  vendor: 'Vente-Privee',
+  dateFormat: 'YYYYMMDD',
+};
+
+const Bill = require('../models/bill');
+const baseUrl = 'https://secure.fr.vente-privee.com';
+
+// Konnector
+const connector = module.exports = baseKonnector.createNew({
+  name: 'Vente-Privee.com',
```
Comment (frankrousseau) [inline server/konnectors/venteprivee.js]: *Vente Privée* would be a better name.

File: server/konnectors/venteprivee.js [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+  slug: 'vente-privee-com',
+  description: 'konnector description vente_privee',
```
Comment (frankrousseau) [inline server/konnectors/venteprivee.js]: don't set description yourself (baseKonnectors does it already).

**Comment by frankrousseau** [comment]
I made a few comments in the code. I'm ok to merge this PR. But can you make the fixes in a new PR?

File: server/konnectors/venteprivee.js (hunk 4 above)
Comment (SomeAverageDev) [inline server/konnectors/venteprivee.js]: how does it make the match between the translation id and the connector ? (is it the slug value, the filename ?)

Comment (frankrousseau) [inline server/konnectors/venteprivee.js]: It uses the cozy-slug library to build the slug then it uses the key `konnector description built_slug`.

File: server/konnectors/venteprivee.js (hunk 1 above)
Comment (SomeAverageDev) [inline server/konnectors/venteprivee.js]: This is the filename for PDF, why not using uppercases ?

**Comment by SomeAverageDev** [comment]
@frankrousseau don't commit the PR yet, i will make the changes

**Comment by frankrousseau** [comment]
Thx @SomeAverageDev

**Comment by frankrousseau** [comment]
Another beautiful konnector into Cozy!

**Comment by SomeAverageDev** [comment]
thanks ;-)

**Pull request merged**